### PR TITLE
CXC-414: layers and preview are larger than one screen height

### PR DESCRIPTION
**Pull request by Vanessi-K**
The height of the preview pop-up and layer-pop-up (when empty) should now not be more than one screen height.

### CODE REVIEW

File: components/ScreenOverlay.vue [hunk 1]
```diff
@@ -48,7 +46,7 @@
     }
 
     .overlay__content {
-        width: 100%;
-        height: 100%;
+        height: calc(100dvh - 3.5rem);
```
Comment (AntoinouM) [inline components/ScreenOverlay.vue:49]: as 3.5 rem is defined as our Nav height, would it be better to have it in a SCSS variable and use it for calculation like this? 
Later on, if we want to change this we would have to manually change all calc

File: pages/editor.vue [hunk 2]
```diff
@@ -345,7 +345,7 @@
 
     .layer-background {
         padding-top: $spacer-5;
-        height: 100dvh;
+        height: calc(100dvh - 3.5rem);
```
Comment (AntoinouM) [inline pages/editor.vue:348]: Following previous comment, I would create a scss variable and use it here.

**Review by AntoinouM — CHANGES_REQUESTED** [review]

**Comment by AntoinouM** [comment]
It works fine for me! I would just suggest a tiny improvement as commented in the code.

File: components/ScreenOverlay.vue (hunk 1 above)
Comment (Vanessi-K) [reply components/ScreenOverlay.vue:49]: Makes total sense

**Pull request merged**